### PR TITLE
Update pmp.yaml

### PR DESCRIPTION
**Pull request by Karthi8883**
Updated RO pump monitoring in canonical and telemetry.

### CODE REVIEW

File: ontology/yaml/resources/HVAC/entity_types/PMP.yaml [hunk 1]
```diff
@@ -119,6 +119,18 @@ PMP_SS_CWCS_VSFC:
   - run_mode
   - lost_power_alarm
 
+PMP_SS_RMM:
+  description: "RO Pump with pressure switch and failed alarm monitoring"
```
Comment (tasodorff) [inline ontology/yaml/resources/HVAC/entity_types/PMP.yaml:123]: can remove RO. is not relevant to the type.

**Review by tasodorff — CHANGES_REQUESTED** [review]
minor change.

File: ontology/yaml/resources/HVAC/entity_types/PMP.yaml (hunk 1 above)
Comment (Karthi8883) [reply ontology/yaml/resources/HVAC/entity_types/PMP.yaml]: Change done

**Review by tasodorff — APPROVED** [review]

**Comment by mschulze17** [comment]
@cstirdivant yours to merge :)

**Pull request merged**